src/main/java/com/sparta/popupstore/domain/promotionevent/repository/PromotionEventRepository.java

```diff
@@ -22,9 +22,13 @@ public interface PromotionEventRepository extends JpaRepository<PromotionEvent,
     Optional<PromotionEvent> findByPromotionEventId(@Param("promotionEventId") Long promotionEventId);
 
     @Modifying
-    @Query(value = "delete PromotionEvent p where timestampdiff(month , p.deletedAt, now()) >= 6", nativeQuery = true)
+    @Query(value = "delete from events e where timestampdiff(month , e.deleted_at, now()) >= 6", nativeQuery = true)
     void deletePromotionEventByDeletedAtAfterSixMonths();
 
+    @Modifying
+    @Query("update PromotionEvent p set p.deletedAt = now() where p.endDateTime <= now() and p.deletedAt is null")
+    void softDeletePromotionEventByTerminated();
```

Comment:
```suggestion
    @Modifying
    @Query("update events e set e.deletedAt = now() where e.endDateTime <= now() and e.deletedAt is null")
    void softDeletePromotionEventByTerminated();
```
DB table 이름을 바꿔서 PromotionEvent는 못 쓸 것 같습니다.

Reply:
그리고 now()로 하는 것 보다는 시간을 입력 받고 그 시간을 이용하는 식으로 하면 어떨까 합니다.

Reply:
저 update 쿼리는 PromotionEvent가 맞습니다 jpql의 경우 테이블명이 아니라 엔티티명으로 해줘야하기 때문에 저희 엔티티 클래스명으로 해줘야합니다 ! 
아래 delete는 nateveQuery로 실제 mysql 쿼리문을 날리기 위해서 events 로 해준겁니다